Fixing cmake to use conan version of RivieraSDK

Michael-Zeimbekakis-Bose opened the pull request.

Michael-Cook-Bose commented on Bose/.gitignore:
```diff
@@ -23,3 +23,4 @@ conaninfo.txt
 activate_run.sh
 deactivate_run.sh
 graph_info.json
+LibMultiThread/builds
```
Please add the final newline (and fix your editor :-))

Michael-Zeimbekakis-Bose replied:
Done

Mark-Hatch-Bose commented:
Any reason we added the txt generator @Michael-Zeimbekakis-Bose, should we explicitly add in MakefilePackage

Michael-Zeimbekakis-Bose commented:
@Mark-Hatch-Bose We dropped the cmake generator with the shift to OpenSourcePackage, so it needed to be added back. The txt generator was already in OpenSourcePackage and I didn't want to override it, so I added it to this specific conanfile.
